web/src/filter.ts

```diff
@@ -808,10 +824,13 @@ export class Filter {
                     };
                 }
             }
-            if (canonicalized_operator === "channels" && operand === "public") {
+            if (
+                canonicalized_operator === "channels" &&
+                ["public", "web-public"].includes(operand)
```

Comment:
Looks like we are utilizing this same array in a lot of places, perhaps seperate it out into some kind of global variable?